ops/inventories/production.yml

```diff
@@ -2,6 +2,6 @@ all:
   children:
     dev:
       hosts:
-        51.75.169.235:
+        51.89.227.206:
```

Comment:
Could you please import my keys in this machine? 🙂

Reply:
It was already the case 😉. Did you try?

Reply:
Yes:

> › ssh debian@51.89.227.206
> debian@51.89.227.206: Permission denied (publickey).

It was because I had a new SSH key, I used an old one to connect and add my new one, this problem is fixed 🙂